modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java

```diff
@@ -495,6 +423,17 @@ private void _putResult(
 		PortalCache<Serializable, Serializable> portalCache = getPortalCache(
 			clazz);
 
+		if (baseModel instanceof CTModel) {
```

Comment:
i don't think this code does anything.  when you put the result it will still end up checking the CTCollectionThreadLocal and putting it in there.

Reply:
oh nevermind... i see that are are returning the original cache instead and not the CTAware one